pkg/userdata/userdata.go

```diff
@@ -122,30 +108,6 @@ func (data *UserData) WriteFiles() (err error) {
 	return nil
 }
 
-// IsBootstrap indicates if the current kubeadm configuration is a master init
-// configuration.
-func (data *UserData) IsBootstrap() bool {
```

Comment:
I think I'd rather see these kept as is. For now it is only kubeadm that decides this but keeping this here has the benefit that we can change the underlying logic without having to be concerned about the `kubeadm` service.